.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+##### Motivation for the changes
+
+<!-- An issue reference or explanation why the change is desirable. -->
+<!-- This speeds up step 1 of the process. -->
+
+
+
+
+
+
+##### Checklist for maintainers
+
+<!-- Contributors: please leave this as is -->
+<!-- Maintainers: tick if completed or explain if not relevant -->
+
+ - [ ] is the idea good? has it been discussed by the Nix team?
```

Comment:
```suggestion
 - [ ] agreed on idea
 - [ ] agreed on implementation strategy
```

To more closely match the [team handbook](https://github.com/NixOS/nix/tree/eaa20f25747eb025c7c4e69fa83f0e455a65393f/maintainers#project-board-protocol).

I thought about writing "approved", but that sounds a lot like bureaucracy.

Reply:
`agreed on idea` &mdash;  `agreed on idea & design` I think the word "design" adds value around here.